code_search/docker/search-index-creator/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:7
```

Comment:
Can't we just use: https://github.com/kubeflow/examples/blob/master/code_search/docker/t2t/Dockerfile.dataflow?
And add to it if needed?